src/qt/coincontroldialog.cpp

```diff
@@ -136,6 +137,13 @@ CoinControlDialog::CoinControlDialog(const PlatformStyle *_platformStyle, QWidge
         sortView(settings.value("nCoinControlSortColumn").toInt(), (static_cast<Qt::SortOrder>(settings.value("nCoinControlSortOrder").toInt())));
 
     GUIUtil::handleCloseWindowShortcut(this);
+
+    if(_model->getOptionsModel() && _model->getAddressTableModel())
```

Comment:
This needs an assert that `_model` isn't `nullptr`.

Reply:
This will crash anyway.

Reply:
assert crashes are marginally easier to debug afaik

Reply:
Yes. But note that the model is received in the constructor and this dialog is instanced in `SendCoinsDialog` which already has the model set.

I'll add the assertion if I had to push again for some reason or if others think the same.